### PR TITLE
Tracing tests for Kafka Broker and Channel

### PR DESCRIPTION
**Pull request by mgencur**
Fixes #[585](https://github.com/knative-sandbox/eventing-kafka-broker/issues/585)

This PR still has [vendor/knative.dev/pkg/test/tracing/traces.go](https://github.com/knative-sandbox/eventing-kafka-broker/compare/main...mgencur:eventing-kafka-broker:broker_tracing_tests_main?expand=1#diff-5c620242fd33f18c3242b639e9eb8ac08720d16249744a3ba1ac8f1d5a9ede42) in vendor directory. It's a copy of [this file from knative/eventing](https://github.com/knative/eventing/blob/main/test/conformance/helpers/tracing/traces.go) with slight changes. I plan to send a PR to knative.dev/pkg and then vendor this file to this repository.
EDIT: Sent https://github.com/knative/eventing/pull/6446 which was merged and this PR has been rebased.

## Proposed Changes

- Enable port-forwarding to Zipkin pod
- Send an event through Kafka Broker 
- Pull the trace from Zipkin
- Verify the Trace

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by mgencur** [comment]
@pierDipi Hi! This one is ready for review. Whenever you have time. Thanks :)

File: include/google/protobuf/any.proto [hunk 1]
```diff
@@ -0,0 +1,158 @@
+// Protocol Buffers - Google's data interchange format
```
Comment (pierDipi) [inline include/google/protobuf/any.proto:1]: I didn't fully get why we have all of these proto files, would you mind expanding?

Comment (mgencur) [reply include/google/protobuf/any.proto:1]: Hi @pierDipi these files got in by running `./hack/update-codegen.sh`. They're probably not related to this PR. Let me rebase and see if this is still required.

Comment (mgencur) [reply include/google/protobuf/any.proto]: OK. The codegen check now passes without those files. So, this PR doesn't include them anymore.

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Comment by mgencur** [comment]
@aliok @pierDipi thanks!

**Pull request merged**